source/wp-content/themes/wporg-developer-2023/inc/block-hooks.php

```diff
@@ -52,33 +51,6 @@ function get_block_content_by_home_url( $block_content, $replacement_home_url =
 	);
 }
 
-/**
- * Filters an article meta block and conditionally removes it.
- *
- * @param string $block_content
- * @param array  $block
- * @return string
- */
-function filter_handbook_meta_link_block( $block_content, $block ) {
-	if ( 'wporg/handbook-meta-link' === $block['blockName'] ) {
-		// Not all handbooks come from GitHub.
-		$local_handbooks = array( 'apis-handbook', 'plugin-handbook', 'theme-handbook' );
-		$post_type = get_post_type();
-
-		if ( in_array( $post_type, $local_handbooks ) ) {
-			return '';
-		}
-
-		// The block editor handbook doesn't have a changelog.
-		// We only know it's the changelog because of the linkURL attribute.
-		if ( 'blocks-handbook' === $post_type && '[article_changelog_link]' === $block['attrs']['linkURL'] ) {
-			return '';
-		}
```

Comment:
This logic replaced by the switch [here](https://github.com/WordPress/wporg-developer/pull/517/files#diff-1cf9dd600383a4c89ba09a878ca8691359b6f6dcece2b9c9b90f3de810f274a4R607)